mapie/classification.py

```diff
@@ -663,13 +670,10 @@ def fit(
         self._check_parameters()
         cv = check_cv(self.cv)
         estimator = self._check_estimator(X, y, self.estimator)
-
         if self.image_input:
             check_input_is_image(X)
-        X, y = check_X_y(
-            X, y, force_all_finite=False, ensure_2d=self.image_input,
-            allow_nd=self.image_input, dtype=["float64", "int", "object"]
-        )
+        X, y = indexable(X, y)
```

Comment:
Does that check that X and y have same `shape[0]` ?

Reply:
Yes